config.go

```diff
@@ -35,12 +35,16 @@ type Config struct {
 	EventsPort int
 	// the interface we should be listening on for events
 	EventsInterface string
+	// dcos token
+	MarathonToken string
 	// HTTPBasicAuthUser is the http basic auth
 	HTTPBasicAuthUser string
 	// HTTPBasicPassword is the http basic password
 	HTTPBasicPassword string
 	// CallbackURL custom callback url
 	CallbackURL string
+	// Token for DCOS environment
```

Comment:
We should mention here that the token will overwrite any basic auth credentials.